src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationInvalidArrayInitIndentWithoutTrailingComments.java

```diff
@@ -0,0 +1,44 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+/**                                                                           //indent:0 exp:0
```

Comment:
remove `//indent:0 exp:0` from this file.

Sad that such bad comments leaked to other files, we can remove them in separate PR.